docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  backend:
+    build: ./backend
+    depends_on:
+      - db
+    ports:
+      - ${PORTS}
+    env_file:
+      - .env
+    container_name: backend
+    command: bash -c "export FLASK_ENV=development
+      && export FLASK_APP=app.py
+      && flask run --host 0.0.0.0 --port 8000"
+    volumes:
+      - ./backend:/backend
+
+  db:
+    image: postgres
+    env_file:
+      - .env
+
+volumes:
+  static-files:
```

Comment:
Where is docker for production?

Reply:
I haven't have nginx yet, so I think it's not necessary to have docker for production

Reply:
It'd be easier for you to have a full dockerized setup from the start